action.yml

```diff
@@ -16,21 +16,77 @@
 
 name: "Semantic release"
 description: "This tool performs semantic release or returns value of next release version"
+
 inputs:
   dry_run:
     description: "dry run - if only version should be returned without releasing new version"
-    default: false
+    default: "false"
     required: false
   extra_plugins:
     description: "plugins beside semantic-release/changelog, semantic-release/git, semantic-release/exec are to be installed"
+  git_committer_name:
+    description: "committer name for semantic release changes"
+    required: false
+  git_committer_email:
+    description: "committer email address for semantic release changes"
+    required: false
+  gpg_private_key:
+    description: "gpg private key matching committer email"
+    required: false
+  passphrase:
+    description: "passphrase for gpg private key"
+    required: false
```

Comment:
Is there a way to declare those as secrets and store them in this repo only, so we don't need to sync them to the repos which need release functionality?

Reply:
Let's leave it lie this for now, anyway the amount of repositories that need this feature is relatively low and I do not see it growing fast in the nearest future.